### DISCUSSION TITLE
Also add "Next / Prev" buttons at the bottom of a commit review

### DISCUSSION THREAD

**Discussion by bgpedersen** [community/community#122101] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When reviewing a Pull Request per commit, you need to click "Next" og "Prev" to go to the next or the previous commit. But the buttons only exists on the top of the page. That means, that you need to manually scroll to the top everytime you are finished with a commit review, to go to the next commit.

If the "Next / Prev" buttons also existed at the end of the commit files review, it would save time and be better UX, because you don't need to scroll up everytime. The natural UX flow is you start the commit -> scroll down through the commit (maybe add comments) and then at the end, you click "Next" to go to the next commit.

Thanks

**Comment by tspeight-optos** [comment]
Duplicate of https://github.com/orgs/community/discussions/69849
